templates/solutions/telco/index.html

```diff
@@ -148,275 +159,489 @@ <h2 class="p-text--small-caps">Canonical is active in key open source initiative
-        <div class="col-3 col-medium-2">
-          <h3 class="p-heading--5">Trusted operating system</h3>
-        </div>
-        <div class="col-6 col-medium-3">
-          <p>Ubuntu is the standard secure Linux for telco workloads running on servers and virtual machines. It offers the widest open source software library and supports the fastest, biggest and most successful digital operations. Ubuntu is also widely tested on the world’s leading silicon to optimise resource consumption.</p>
-          <p>66% of experienced developers prefer Ubuntu. Chiselled Ubuntu images drive optimal and minimally-sized containers.</p>
-          <hr class="p-rule" />
-          <p>
-            <a href="https://ubuntu.com/telco">Learn more about Ubuntu for telco&nbsp;&rsaquo;</a>
-          </p>
-        </div>
+      <div class="col">
+        <h2>Enabling 5G adoption with industry leaders</h2>
       </div>
-      <hr class="p-rule" />
-      <div class="row p-section--shallow">
-        <div class="col-3 col-medium-2">
-          <h3 class="p-heading--5">Optimal virtualisation infrastructure</h3>
-        </div>
-        <div class="col-6 col-medium-3">
-          <p>Build your own cloud and use powerful automation tools to lower your maintenance burdens and Total Cost of Ownership (TCO).</p>
-          <p>Choose Canonical OpenStack for data centres or Microcloud for small private clouds and edge environments. Running on top of both, Canonical Kubernetes provides repeatable and trusted deployment and operations of CNFs.</p>
+      <div class="col">
+        <div class="u-align--center u-embedded-media u-hide--small u-hide--medium">
+          <iframe src="https://www.youtube.com/embed/0oFnOHJrRQs?si=tVLHY2fIyBq_1B8B"
+                  class="u-embedded-media__element"
+                  title="YouTube video player"
+                  frameborder="0"
+                  allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share"
+                  referrerpolicy="strict-origin-when-cross-origin"
+                  allowfullscreen></iframe>
         </div>
+        <p>See how BT and Canonical deliver 5G to UK stadiums</p>
+        <p>
+          Canonical worked with BT Group and its consumer division, EE, to drive better in-stadium connectivity across the UK as part of a larger 5G roll out strategy.
+        </p>
+        <hr class="p-rule" />
+        <p>
+          <a href="/blog/bt-group-and-canonical-deliver-5g-to-uk-stadiums ">Read the story&nbsp;&rsaquo;</a>
+        </p>
       </div>
+    </div>
+  </section>
+
+  <section>
+    <div class="row">
       <hr class="p-rule" />
-      <div class="row">
-        <div class="col-3 col-medium-2">
-          <h3 class="p-heading--5">Full security coverage</h3>
+      <div class="col-3 col-medium-6">
+        <p class="p-text--small-caps">Trusted by telco vendors around the world</p>
+      </div>
+      <div class="col-9 col-medium-6">
```

Comment:
```suggestion
      <div class="col-9 col-medium-6 p-section">
```
This will give the appropriate spacing below